mrp_unbuild_cust_alu_analytics_maintenance/views/mrp_unbuild_incidence_views.xml

```diff
@@ -15,4 +15,18 @@
         </field>
     </record>
 
+    <record id="mrp_unbuild_incidence_tree_inherit_view" model="ir.ui.view">
```

Comment:
As added to tree view, I should add it to `search` view as well (searching for this file, and grouping too).